src/Bridge.jl

```diff
@@ -261,12 +261,24 @@ unfame(fo::FameObject{:series,:precision,FR}) where {FR} =
     TSeries(_date_to_mit(FR, fo.first_index[]), _unmissing!(fo.data, Val(:precision)))
 unfame(fo::FameObject{:series,:numeric,FR}) where {FR} =
     TSeries(_date_to_mit(FR, fo.first_index[]), _unmissing!(fo.data, Val(:numeric)))
-unfame(fo::FameObject{:series,:boolean,FR}) where {FR} =
-    TSeries(_date_to_mit(FR, fo.first_index[]), [d != 0 for d in fo.data])
-unfame(fo::FameObject{:series,TY,FR}) where {TY,FR} =
-    TSeries(_date_to_mit(FR, fo.first_index[]), [_date_to_mit(TY, d) for d in fo.data])
 unfame(fo::FameObject{:series,:string,FR}) where {FR} =
     copy(fo.data) # error("Can't handle string series")
+function unfame(fo::FameObject{:series,TY,FR}) where {TY,FR}
+    if length(fo.data) == 1 && _ismissing(fo.data[1], Val(:FR))
```

Comment:
I think it should be `Val(FR)` not `VAL(:FR)`

Reply:
Thanks for catching that. It turns out it was still working because the value doesn't get used, it just gets routed to the date-checking `_ismissing` which accepts all the values that aren't in the other `_ismissing` signatures.

I changed it to `Val(TY)` since we're technically checking against the data type in the TSeries, which can be different in than the frequency of the series itself. Neither value gets used, but it's still good to be consistent.

Reply:
Indeed, you're right!